src/HelixProject.js

```diff
@@ -163,6 +163,14 @@ class HelixProject {
     return this._server;
   }
 
+  /**
+   * Returns the index resource path: /index or /README
+   */
+  get index() {
```

Comment:
why no just return `this._index` ?

Reply:
Linting complains with `this._cfg._index` (2x underscores). And it is nicer to have a method. 
If the comment is about the content of the method, then yes I can compute a `_index` and just return it.

Reply:
:-)

no, I mean, why do you take the substring of indexMd?